sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -149,11 +149,8 @@ statement
     | ALTER (TABLE | VIEW) multipartIdentifier
         UNSET TBLPROPERTIES (IF EXISTS)? tablePropertyList             #unsetTableProperties
     | ALTER TABLE multipartIdentifier
```

Comment:
we can distinguish the two `multipartIdentifier`
```
ALTER TABLE table=multipartIdentifier ... COLUMN? column=multipartIdentifier
```

You can fix it in your followup

Reply:
ok.